OceanLab/eof.py

```diff
@@ -121,3 +121,162 @@ def my_eof_interp(M,nmodes,errmin=1e-15,repmax=None):
     vi = (M.T+Mmean).T
 
     return vi
+
+#=========================================
+# PERFORM COMPLEX EOF
+#=========================================
+from scipy.signal import hilbert
+import scipy.linalg as la
```

Comment:
Imports should be placed in the top of each file regardless of the fuction in which they are used

Reply:
Done